lib/private/shell.ts

```diff
@@ -67,6 +64,27 @@ export async function shell(command: string[], options: ShellOptions = {}): Prom
   });
 }
 
+function handleShellOutput(
+  chunk: Buffer | string,
+  options: ShellOptions,
+  eventType: ShellEventType
+): void {
+  switch (options.subprocessOutputDestination) {
+    case 'ignore':
+      return;
+    case 'publish':
+      options.shellEventPublisher(eventType, chunk.toString('utf-8'));
+      break;
+    case 'stdio':
+    default:
+      if (eventType == 'data_stdout' || eventType == 'open') {
```

Comment:
I'm still not convinced that the `open` event used to be sent to `stdout` in the old code.

How sure are you about that?

Reply:
good catch

Reply:
It went nowhere I'm pretty sure?